app/frontend/src/styles/application/jobseekers/jobseeker.scss

```diff
@@ -1,6 +1,8 @@
 .jobseeker {
   .create-alert__button {
-    text-align: right;
+    .govuk-button {
+      margin-bottom: govuk-spacing(1);
```

Comment:
instead of this could you not just put an inline govuk style on the element in template? `class="govuk-!-margin-bottom-1"` i think thats better for a style like this that is very specific to one instance of something on one page